docs/README.configuration.md

```diff
@@ -19,7 +19,10 @@ ipv6_mtu = 1500,
 policy_icmpv4_incoming = policies['ALLOW'],
 policy_icmpv6_incoming = policies['ALLOW'],
 policy_icmpv4_outgoing = policies['ALLOW'],
-policy_icmpv6_outgoing = policies['ALLOW']
+policy_icmpv6_outgoing = policies['ALLOW'],
+v4_vlan_tag = C.htonl(0x81000444),
+v6_vlan_tag = C.htonl(0x81000666),
+vlan_tagging = true
```

Comment:
A couple of thoughts, none of them a blocker for merging this PR:

* How about keeping always the trailing comma (or semicolon) in cases like this where the assignments are part of a table, so patches to them do not have an “add a comma/semicolon to an existing line” part?
* For configuraion files, we could finish the lines with semicolons, taking advantage of Lua allowing both for separating items in tables. For this case where we end with something which looks like a line-oriented configuration file format, semicolons look better in the eye than commas (IMHO) because they are more commonly used as line terminators.